python/shotgun_model/simple_shotgun_model.py

```diff
@@ -70,6 +70,7 @@ def load_data(
                   is handling, allowing a user to for example show the twenty most recent notes or
                   similar.
         :param columns: List of Shotgun fields to use to populate the model columns
+        :param editable_columns: A subset of ``columns`` that will be editable in views that use this model.
```

Comment:
if we add examples to the main `_load_data` method, we could have a sphinx reference from this method back to the main one.